code/API_definitions/Edge-Application-Management.yaml

```diff
@@ -547,6 +547,59 @@ paths:
           $ref: '#/components/responses/500'
         '503':
           $ref: '#/components/responses/503'
+  /clusters:
+    get:
+      security:
+        - openId:
+            - edge-application-management:clusters:read
+      tags:
+        - Cluster
+      summary: Retrieve a list of the available clusters
+      description: |
+        List available cluster information
+      operationId: getClusters
+      parameters:
+        - $ref: '#/components/parameters/x-correlator'
+        - name: region
+          description: |
+            Human readable name of the geographical Edge Cloud Region of
+            the Cluster. Defined by the Edge Cloud Provider.
+          in: query
+          required: false
+          schema:
+            $ref: '#/components/schemas/EdgeCloudRegion'
+        - name: clusterRef
+          description: |
+            A globally unique identifier for the Cluster.
+          in: query
+          required: false
```

Comment:
Is it correct that if the clusterRef is not present in query than the response will include information of all the clusters in that edge cloud zone and/or region. If it is correct then we can include this indication in description. To me we need this capability as there is no other method by which I as a developer can retrieve the clusterRef.

Reply:
Yes, that's the intent, as stated in the summary. I'd argue it's somewhat obvious since these query parameters are optional, but I can make it more clear in the summary.